frontend/src/patientApp/timeOfTest/__snapshots__/AoEPatientFormContainer.test.tsx.snap

```diff
@@ -517,7 +517,7 @@ exports[`AoEPatientFormContainer snapshot 1`] = `
               aria-required="false"
               className="usa-input"
               id="text-test-1"
-              max="2021-02-25"
+              max="2021-02-26"
```

Comment:
Huh?

Reply:
Good question 🤔

Reply:
Ah, max date is 'today' & needs to be set explicitly in the snapshot test.

Reply:
Fixed!